validator/remote/src/main/java/tech/pegasys/teku/validator/remote/RemoteValidatorApiHandler.java

```diff
@@ -281,7 +282,12 @@ public SafeFuture<Optional<BlockContainer>> createUnsignedBlock(
 
   @Override
   public SafeFuture<SendSignedBlockResult> sendSignedBlock(
-      final SignedBlockContainer blockContainer) {
+      final SignedBlockContainer blockContainer,
+      final Optional<BroadcastValidationLevel> broadcastValidationLevel) {
+    // we are not going to use V2 to send blocks, so we can ignore it
```

Comment:
The comment is actually wrong. We could move to V2 if V1 will be deprecated but I don't think we will send a value there (broadcast_validation is an optional field) since the usage of the param isn't really meant to be used by vanilla VCs.

Reply:
Ok, worth changing a comment

Reply:
yep I did